### PR TITLE
alipay finished

### PR DESCRIPTION
**Pull request by Michael-Xuke**

### CODE REVIEW

File: alipay_app/views.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from django.shortcuts import redirect
+
+from Alipay import settings
+
+# Create your views here.
+
+
+def alipay(request):
+    alipay = AliPay(
+        appid="2016092100560320",
```
Comment (Michael-Xuke) [inline alipay_app/views.py]: appid硬编码,不太好.

File: .idea/workspace.xml [hunk 2]
```diff
@@ -26,6 +83,17 @@
   <component name="Git.Settings">
     <option name="RECENT_GIT_ROOT_PATH" value="$PROJECT_DIR$" />
   </component>
+  <component name="IdeDocumentHistory">
+    <option name="CHANGED_PATHS">
+      <list>
+        <option value="$PROJECT_DIR$/Alipay/settings.py" />
+        <option value="$PROJECT_DIR$/Alipay/urls.py" />
+        <option value="$PROJECT_DIR$/alipaykeys/app_private_key.pem" />
+        <option value="$PROJECT_DIR$/alipay_app/views.py" />
```
Comment (gcxcode) [inline .idea/workspace.xml]: nice

File: .idea/workspace.xml [hunk 3]
```diff
@@ -11,11 +14,65 @@
...
-            <state relative-caret-position="136">
-              <caret line="8" column="12" lean-forward="false" selection-start-line="8" selection-start-column="12" selection-end-line="8" selection-end-column="12" />
+            <state relative-caret-position="251">
+              <caret line="124" column="80" lean-forward="false" selection-start-line="124" selection-start-column="65" selection-end-line="124" selection-end-column="80" />
+              <folding />
+            </state>
+          </provider>
+        </entry>
+      </file>
+      <file leaf-file-name="urls.py" pinned="false" current-in-tab="false">
```
Comment (currynihao) [inline .idea/workspace.xml]: hahaahh

File: alipay_app/views.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
+from alipay import AliPay
+from django.shortcuts import redirect
+
+from Alipay import settings
+
+# Create your views here.
+
+
+def alipay(request):
+    alipay = AliPay(
```
Comment (Gin79) [inline alipay_app/views.py]: 代码太骚了

File: .idea/workspace.xml [hunk 5]
```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ChangeListManager">
-    <list default="true" id="03c36f20-4d5f-433f-898c-709848ea9d68" name="Default" comment="" />
+    <list default="true" id="03c36f20-4d5f-433f-898c-709848ea9d68" name="Default" comment="">
+      <change type="MODIFICATION" beforePath="$PROJECT_DIR$/.idea/workspace.xml" afterPath="$PROJECT_DIR$/.idea/workspace.xml" />
+      <change type="MODIFICATION" beforePath="$PROJECT_DIR$/Alipay/settings.py" afterPath="$PROJECT_DIR$/Alipay/settings.py" />
```
Comment (currynihao) [inline .idea/workspace.xml]: hahahhahaha

File: .idea/workspace.xml [hunk 6]
```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ChangeListManager">
-    <list default="true" id="03c36f20-4d5f-433f-898c-709848ea9d68" name="Default" comment="" />
+    <list default="true" id="03c36f20-4d5f-433f-898c-709848ea9d68" name="Default" comment="">
+      <change type="MODIFICATION" beforePath="$PROJECT_DIR$/.idea/workspace.xml" afterPath="$PROJECT_DIR$/.idea/workspace.xml" />
```
Comment (wuruibo0620) [inline .idea/workspace.xml]: nice

File: alipay_app/views.py (hunk 4 above)
Comment (UnionL) [reply alipay_app/views.py]: 6666

File: .idea/workspace.xml [hunk 7]
```diff
@@ -11,11 +14,65 @@
...
+      <file leaf-file-name="settings.py" pinned="false" current-in-tab="false">
         <entry file="file://$PROJECT_DIR$/Alipay/settings.py">
           <provider selected="true" editor-type-id="text-editor">
-            <state relative-caret-position="136">
-              <caret line="8" column="12" lean-forward="false" selection-start-line="8" selection-start-column="12" selection-end-line="8" selection-end-column="12" />
+            <state relative-caret-position="251">
+              <caret line="124" column="80" lean-forward="false" selection-start-line="124" selection-start-column="65" selection-end-line="124" selection-end-column="80" />
+              <folding />
+            </state>
+          </provider>
```
Comment (XiaoLing-git) [inline .idea/workspace.xml]: gdcfgdgsdfgsd

**Pull request merged**